packages/create-contentful-app/template/.contentfulrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this necessary?

Reply:
It will crash if `contentful login` is trying to parse JSON.